tests/behat/features/apply-formatting.feature

```diff
@@ -0,0 +1,29 @@
+@retry @job1
```

Comment:
Remove all of the `@job1` annotations from all the behat files

Reply:
Why?

Reply:
Unnecessary, and confused me thinking there were two jobs, when there is only one

Reply:
Done